docs/meshstack.monitoring.md

```diff
@@ -24,6 +24,63 @@ These email inboxes be configured in the [meshStack configuration model](meshsta
 }
 ```
 
+## Health endpoint
+
+meshStack configures a health endpoint which returns the current status and also the status of database connections. These information are available under path `/actuator/health`.
+
+Example:
+
+```json
+{
+    "status": "UP",
+    "components": {
+        "db": {
+            "status": "UP",
+            "details": {
+                "database": "MariaDB",
```

Comment:
d: Hmm, perhaps it would be ciritical to provide info that we use mariadb via a public endpoint. Can the details be removed?

Reply:
I know. But I couldn't find any confg to return only the status. Then we need to implement a custom indicator.

Reply:
I couldn't find a soultion to return only the status of the db connection.
How should we handle this now? @sttomm

Reply:
@chewbeccy Do you think its ok that we also return the db name? Or is this a security issue?

Reply:
Not a fan to be honest of sharing DB names publicly. I do see it as a medium risk and would like to mitigate it. But at the moment you dont see any option hiding it?

Reply:
Currently I expose the db status only with a specific actuator configuration. If we want to hide the db name then we need to implement a custom indicator to expose only the db status. I'm not sure how big the effort is to implement a custom db status indicator.